pkgs/development/python-modules/rctclient/default.nix

```
@@ -0,0 +1,34 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  click,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "rctclient";
+  version = "0.0.4";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "svalouch";
+    repo = "python-rctclient";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-QPla5h8wSM9Ynj44Uwc1a2yAnu8TXbyBWzVHQeW6jnI=";
+  };
+
+  build-system = [ setuptools ];
+
+  optional-dependencies.cli = [ click ];
+
+  pythonImportsCheck = [ "rctclient" ];
```

```suggestion
  pythonImportsCheck = [ "rctclient" ];

  nativeCheckInputs = [ pytestCheckHook ];
```

Enable the tests for this package.